<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SNMP_PDU_Viewer</class>
 <widget class="QWidget" name="SNMP_PDU_Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="LE_RequestID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Request ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_ErrorStatus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Error Status</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_ErrorIndex">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Error Index</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SPN_VarBinding">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>44</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Variable Bindings</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Num Bindings</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_NumBindings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="SNMP_VarBinding_Viewer" name="PGE_VarBindings" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>591</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SNMP_VarBinding_Viewer</class>
   <extends>QWidget</extends>
   <header>snmp_varbinding_viewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
